source/patterns/@aws-solutions-constructs/aws-s3-step-function/test/s3-step-function.test.ts

```diff
@@ -61,18 +44,10 @@ test('override eventRuleProps', () => {
     eventRuleProps: {
       eventPattern: {
         source: ['aws.s3'],
-        detailType: ['AWS API Call via CloudTrail'],
+        detailType: ['Object Created'],
```

Comment:
This defines the single event we're currently worried about, correct?

Reply:
Yes